rocksdb/options/dbopts.nim

```diff
@@ -111,6 +111,9 @@ proc defaultDbOptions*(autoClose = false): DbOptionsRef =
   # Enable creating column families if they do not exist
   dbOpts.createMissingColumnFamilies = true
 
+  # Make sure flush is atomic accross column families
+  dbOpts.atomicFlush = true
```

Comment:
this is not something we want to enable by default, ie it carries greater (performance) implications than is obvious

Reply:
Ok thanks, will remove